tests/index.test.js

```diff
@@ -251,16 +348,16 @@ describe('Options', () => {
     expect(console.log).toHaveBeenCalledTimes(5)
     expect(console.log).toHaveBeenCalledWith(green('copied:'))
     expect(console.log).toHaveBeenCalledWith(
-      green(`  ${bold('src/assets/asset-1.js')} → ${bold('dist/asset-1.js')}`)
+      green(`  ${bold('src/assets/asset-1.js')} → ${bold(join('dist', 'asset-1.js'))}`)
```

Comment:
Interesting, so it outputs backslashes on windows 🤔

I'll try to resolve this in `src` code so paths won't be different in mac/windows since this doesn't match with `target.src` ones.